tests/scripts/pulp_rpm/test_uln_remote.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "rpm" || exit 3
```

Comment:
This probably needs a version specifier for the rpm plugin.

Reply:
Added the minimum version.